### DISCUSSION TITLE
Faraday::Logging::ColorFormatter

### DISCUSSION THREAD

**Discussion by kobusjoubert** [lostisland/awesome-faraday#19] [Show and tell]
Not sure if you guys would be interested in adding this to your list, but here it is anyways. It's just a logger color formatter to display the requests being made in an ActiveRecord style color way.

- https://rubygems.org/gems/faraday-logging-color_formatter
- https://github.com/kobusjoubert/faraday-logging-color_formatter

![](https://user-images.githubusercontent.com/3071529/212807978-609ee381-f445-4d98-9994-31f7a4e5caa5.png)

**Comment by iMacTia** [comment]
This is pretty cool!
I'm gonna add a new section to the README for tooling 😄, this will be the first entry 🎉 !

**Reply by kobusjoubert** [reply]
Awesome!!! Thanks iMacTia!!!! 🥳
